### PR TITLE
I name everything EEEE nowadays

### PR DESCRIPTION
**Pull request by informathemusic**
Even my debug statements are eee, even in [Last Barrier](https://lb.imm.codes/) (see console)

### CODE REVIEW

File: web/chatblocks.js [hunk 1]
```diff
@@ -2206,6 +2206,51 @@ Blockly.JavaScript["utility_post_http"] = function(block) {
...
+};
+
+Blockly.JavaScript["utility_post_http_get_rslt"] = function(block) {
+	var value_url = Blockly.JavaScript.valueToCode(block, "URL", Blockly.JavaScript.ORDER_ATOMIC);
+	var code = `await (await fetch( ${value_url}, {
+		"method": "POST",
+		"headers": {
+			"Content-Type": "application/json"
+		},
+		"body": ${value_value}
```
Comment (daanbreur) [inline web/chatblocks.js:2248]: You didnt make a variable value_value!

File: web/chatblocks.js [hunk 2]
```diff
@@ -2206,6 +2206,51 @@ Blockly.JavaScript["utility_post_http"] = function(block) {
...
+		"helpUrl": ""
+	  });
+	  this.setColour(35);
+	  this.setTooltip("");
+	  this.setHelpUrl("https://www.instafluff.tv");
+	}
+};
+
+Blockly.JavaScript["utility_post_http_get_rslt"] = function(block) {
+	var value_url = Blockly.JavaScript.valueToCode(block, "URL", Blockly.JavaScript.ORDER_ATOMIC);
```
Comment (daanbreur) [inline web/chatblocks.js:2242]: Insert `value_value` here!

**Review by daanbreur — CHANGES_REQUESTED** [review]
Fix my 2 comments and it should be alright!!

**Comment by informathemusic** [comment]
Eh
nvm about it

**Comment by daanbreur** [comment]
Okey!
Then ill close this PR. reopen it when needed

**Comment by daanbreur** [comment]
Okey!
im going to close this PR then, reopen when needed.

**Pull request closed without merging**